### ISSUE TITLE
Ruby 2.0.0, EventMachine 1.0.4: Assertion `nbytes > 0' failed

### ISSUE THREAD

**Issue by alexnorthsoul** [eventmachine/eventmachine#559]
We've been facing this issue using 1.0.3 and 1.0.4

ruby: ed.cpp:1025: void ConnectionDescriptor::_WriteOutboundData(): Assertion `nbytes > 0' failed

```
# ruby -v
ruby 2.0.0p598 (2014-11-13 revision 48408) [x86_64-linux]
```

**Comment by sodabrew** [comment]
Please provide a backtrace

**Comment by tmm1** [comment]
Try `EM.epoll=true` before `EM.run`

**Comment by alexnorthsoul** [comment]
@sodabrew, unfortunately there is no full backtrace yet.
@tmm1, thank you for hint, I'll try this within 24 hours

**Comment by sodabrew** [comment]
Ping @alexnorthsoul Any luck reproducing again / or resolution with today's release?

**Comment by sodabrew** [comment]
This couldn't be #51 because there's no kqueue on

**Comment by tmm1** [comment]
This was probably fixed in #525
